### PR TITLE
Nick and Victoria's Sass-101 HW

### PR DESCRIPTION
**Pull request by vccalhoun14**

### CODE REVIEW

File: Nick_Victoria/css/app.css [hunk 1]
```diff
@@ -0,0 +1,336 @@
+/*  SECTIONS  ============================================================================= */
```
Comment (blackfalcon) [inline Nick_Victoria/css/app.css:1]: You need to delete the file from the repo post updating the gitignore in order to remove it from the repo.

File: Nick_Victoria/css/app.css.map [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (blackfalcon) [inline Nick_Victoria/css/app.css.map:1]: delete from repo

File: Nick_Victoria/sass/_config.scss [hunk 3]
```diff
@@ -0,0 +1,14 @@
+$mobile: 'screen and (max-width: 30em)';
+$tablet: 'screen and (max-width: 1024px)';
+
+@mixin breakpoint ($media) {
```
Comment (blackfalcon) [inline Nick_Victoria/sass/_config.scss]: this is a mixin, not a config.

File: Nick_Victoria/sass/base/_color.scss [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+$background-color: $black;
+$link-color: $white;
+$header-color: $guardsman-red;
+
+$nav-background-color: $guardsman-red;
+
+$contact-text: $limeade;
+
+$hover-box-border-color: $white;
+$hover-box-shadow-color: $mercury;
```
Comment (blackfalcon) [inline Nick_Victoria/sass/base/_color.scss]: from here up, I would say that these are config values.

File: Nick_Victoria/sass/base/_color.scss [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+
+// Global Color Values
```
Comment (blackfalcon) [inline Nick_Victoria/sass/base/_color.scss:1]: From here down, these are typography rules.

File: Nick_Victoria/sass/base/responsive/_3cols.scss [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+}
+
+.span_1_of_3 {
+  width: 32.26%;
+}
+
+
+/*  GO FULL WIDTH AT LESS THAN 480 PIXELS */
+
+@media only screen and (max-width: 480px) {
```
Comment (blackfalcon) [inline Nick_Victoria/sass/base/responsive/_3cols.scss:18]: why not use the media query mixin here?

File: Nick_Victoria/sass/components/_contact.scss [hunk 7]
```diff
@@ -0,0 +1,23 @@
+.contact {
+  text-align: center;
+  color: $contact-text;
+
+  & > h1 {
```
Comment (blackfalcon) [inline Nick_Victoria/sass/components/_contact.scss]: In this use case, the `&` is not adding value.

File: Nick_Victoria/sass/components/_footer.scss [hunk 8]
```diff
@@ -0,0 +1,45 @@
+.site-footer {
+  font-size: 1em;
+  border-top: solid 0.15em #ccc;
+  padding: 0.25% 0 0 0;
+  margin: 0;
+  p{
```
Comment (blackfalcon) [inline Nick_Victoria/sass/components/_footer.scss]: fix `p{` => `p {`

File: Nick_Victoria/style/app.css [hunk 9]
```diff
@@ -0,0 +1,335 @@
+/*  SECTIONS  ============================================================================= */
```
Comment (blackfalcon) [inline Nick_Victoria/style/app.css:1]: delete from repo

File: Nick_Victoria/style/app.css.map [hunk 10]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (blackfalcon) [inline Nick_Victoria/style/app.css.map:1]: delete from repo

**Comment by blackfalcon** [comment]
Please update or I will grade as is

File: Nick_Victoria/sass/base/responsive/_3cols.scss (hunk 6 above)
Comment (nklunder) [inline Nick_Victoria/sass/base/responsive/_3cols.scss:18]: We just left this as is from the responsive grid page. We can change if needed.

**Pull request closed without merging**